File: parse_pdf.py
import fitz
import sys

doc = fitz.open('public/KARTIK MULTI SOLUTIONS- website structure.pdf')
text = ""
for page in doc:
    text += page.get_text()

with open('parsed-doc.txt', 'w', encoding='utf-8') as f:
    f.write(text)

File: parse_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse/lib/pdf-parse.js'); // check if this works

let dataBuffer = fs.readFileSync('public/KARTIK MULTI SOLUTIONS- website structure.pdf');

pdf(dataBuffer).then(function(data) {
    console.log(data.text);
}).catch(function(err) {
    console.error(err);
});
